pynumaflow/info/info_server.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from pynumaflow import __version__
+from pynumaflow.info.info_types import ServerInfo, EOF
+import json
+
+
+def get_sdk_version() -> str:
+    """
+    Return the pynumaflow SDK version
+    """
+    return __version__
+
+
+def write(serv: ServerInfo, info_file):
+    """
+    Write the ServerInfo to a file , shared with the client (numa container).
+
+    args:
+        serv: The ServerInfo object to be shared
+        info_file: the shared file path
+    """
+    try:
+        data = serv.__dict__
+        with open(info_file, "w+") as f:
```

Comment:
`w` is good enough, right?

Reply:
We just need to write to the file here right? Should be good for that